### PR TITLE
WIP: webpack

### PR DESCRIPTION
**Pull request by davidmurdoch**
Draft, look away.

### CODE REVIEW

**Comment by davidmurdoch** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,19 @@
 {
   "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
+  "editor.formatOnSave": false,
```
Comment (brad-decker) [inline .vscode/settings.json:3]: This should be opt out per person, not as a org wide setting. I quite like formatOnSave on.

Comment (davidmurdoch) [reply .vscode/settings.json:3]:
```suggestion
```

**Comment by davidmurdoch** [comment]
closing in favor of https://github.com/MetaMask/metamask-extension/pull/22506

**Pull request closed without merging**